examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -111,7 +111,7 @@ class MenuDemoState extends State<MenuDemo> {
             primary: new Text('An item with a sectioned menu'),
             right: new PopupMenuButton<String>(
               onSelected: showMenuSelection,
-              items: <PopupMenuItem>[
+              items: <PopupMenuEntry<String>>[
                 new PopupMenuItem(
```

Comment:
Sorry yes you're right.  All of the menu items in this example should be PopupMenuItem<String>.  Will fix that.